bot/exts/evergreen/latex.py

```diff
@@ -69,6 +80,9 @@ async def latex(self, ctx: commands.Context, *, text: str) -> None:
 
             await ctx.send(file=discord.File(image, "latex.png"))
 
+            with open(image_path, "wb") as f:
+                f.write(image.getbuffer())
```

Comment:
Could we move this into the `_render` function, since it's blocking? Could be worth renaming the function or updating the docstring after too, to reflect it also saves the file.